src/constants/oracleConfig.ts

```diff
@@ -69,12 +82,28 @@ if (process.env.REACT_APP_PROVIDER_URL_288) {
   };
 }
 
+if (process.env.REACT_APP_PROVIDER_URL_5) {
+  const config: oracle.types.state.PartialChainConfig = {
+    rpcUrls: [process.env.REACT_APP_PROVIDER_URL_5],
+    nativeCurrency: CHAINS[goerliChainId].nativeCurrency,
+    chainName: CHAINS[goerliChainId].name,
+    blockExplorerUrls: [CHAINS[goerliChainId].explorerUrl],
+  };
+  optimisticChains[goerliChainId] = {
+    ...config,
+    // sdk currently does not have a default address for chain 5, so manually provide in config
+    optimisticOracleAddress: "0x6f26Bf09B1C792e3228e5467807a900A503c0281",
```

Comment:
Do we also need to specify the V2 address (0xA5B9d8a0B0Fa04Ba71BDD68069661ED5C0848884) somewhere?

Reply:
There are fallbacks in the sdk in case you dont want to specify it, i just didnt have the v1 address in when i published the sdk, but did have the v2 address